src/main/java/com/google/api/generator/gapic/composer/defaultvalue/DefaultValueComposer.java

```diff
@@ -346,18 +347,39 @@ public static Expr createSimplePagedResponse(
             .setMethodName("setNextPageToken")
             .setArguments(ValueExpr.withValue(StringObjectValue.withValue("")))
             .build();
-    pagedResponseExpr =
-        MethodInvocationExpr.builder()
-            .setExprReferenceExpr(pagedResponseExpr)
-            .setMethodName(String.format("addAll%s", JavaStyle.toUpperCamelCase(repeatedFieldName)))
-            .setArguments(
-                MethodInvocationExpr.builder()
-                    .setStaticReferenceType(
-                        TypeNode.withReference(ConcreteReference.withClazz(Arrays.class)))
-                    .setMethodName("asList")
-                    .setArguments(responseElementVarExpr)
-                    .build())
-            .build();
+    if (isMap) {
+      pagedResponseExpr =
```

Comment:
Ditto with the sample code, since this is longer and has some nested nodes.

Reply:
Added two code snippets (one for maps, one for arrays)